elements/lisk-p2p/src/errors.ts

```diff
@@ -84,9 +84,23 @@ export class InvalidPeerError extends Error {
 }
 
 export class RequestFailError extends Error {
-	public constructor(message: string) {
+	public peerId: string;
+	public peerVersion: string;
+	public errorResponse: Error;
+	public constructor(
+		message: string,
+		errorResponse?: Error,
+		peerId?: string,
```

Comment:
If we use `peerId` here, then we should also use `peerId` in `RPCResponseError`.

Reply:
Yes, it should use peerId as its more precise, I'll update it for RPC response as well